packages/react/src/components/modal/modal.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import type { Meta } from "@storybook/react";
+import { ModalArgs, modalMeta, modalStories } from "dso-toolkit";
+
+import { templateContainer } from "../../templates";
+
+import readme from "./readme.md?raw";
+
+import { decorator } from "./modal.decorator";
+import {
+  activeBody,
+  activeFooter,
+  confirmBody,
+  confirmFooter,
+  passiveBody,
+  passiveFooter,
+  loadingBody,
+  datePickerBody,
+} from "./modal.content";
+
+const meta: Meta<ModalArgs> = {
+  ...modalMeta({ readme }),
+  title: "Modal",
+};
+
+export default meta;
+
+const { Passive, Confirm, Fullscreen, Active, WithDatepicker, Loading } = modalStories({
+  templateContainer,
+  storyTemplates: (templates) => {
+    const { modalTemplate } = templates;
+
+    return {
+      modalTemplate,
+      activeBody,
+      activeFooter,
+      passiveBody,
+      passiveFooter,
+      confirmBody,
+      confirmFooter,
+      loadingBody: loadingBody(templates),
+      datePickerBody: datePickerBody(templates),
+    };
+  },
+  decorator,
```

Comment:
De decorator is alleen nodig voor het HTML/CSS component. Het React component maakt gebruik van het Web Component.

Reply:
decorators: [decorator ? (story) => decorator(story) : (story) => story],